middleware/afc_status.py

```
@@ -0,0 +1,209 @@
+    except requests.ConnectionError:
+        logger.debug("AFC status: Moonraker not reachable")
+        return None
+    except requests.Timeout:
+        logger.warning("AFC status: Moonraker request timed out")
+        return None
+    except requests.HTTPError as e:
+        # 404 likely means AFC is not installed
+        if e.response is not None and e.response.status_code == 404:
+            logger.warning("AFC status: endpoint not found — AFC may not be installed")
+        else:
+            logger.error(f"AFC status: HTTP error: {e}")
```

Ruff (0.15.6)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)